### PR TITLE
ppc64le add Debian 9

### PR DESCRIPTION
**Pull request by vladbogo**
Add Debian 9 for ppc64le

### CODE REVIEW

**Review by fauust — APPROVED** [review]
Good!
Maybe you can squash the 4 commit in one (it is also ok like this IMO).

File: .github/workflows/bb_containers.yml [hunk 1]
```diff
@@ -103,13 +100,12 @@ jobs:
...
-            echo "TAG=rhel8" >>$GITHUB_ENV
-          else
-            echo "TAG=${TAG_TMP/:/}" >>$GITHUB_ENV
-          fi
+          case $TAG_TMP in
+            "quay.io/centos/centos:stream8") echo "TAG=centos-stream8" >>$GITHUB_ENV;;
+            "ubi8") echo "TAG=rhel8" >>$GITHUB_ENV;;
+            "ppc64le/debian:9") echo "TAG=debian9-ppc64le" >>$GITHUB_ENV;;
+            *) echo "TAG=${TAG_TMP/:/}" >>$GITHUB_ENV;;
+          esac
```
Comment (fauust) [inline .github/workflows/bb_containers.yml:108]: I am pedantic but bellow you have put a space before `;;` and I believe its cleaner (and it follows https://google.github.io/styleguide/shellguide.html#s5.5-case-statement).

**Comment by grooverdan** [comment]
I'm assuming "[Change tag name for ppc64le/debian:9](https://github.com/MariaDB/mariadb.org-tools/pull/113/commits/257fb1f16de7ab712d287575ee61002a5de6b4c2)" is just for testing as rework of the bb config would be need for it.

**Comment by vladbogo** [comment]
It is not. The ppc64le Debian 9 base image needs to be pulled from a different registry _ppc64le/debian:9_ instead of _debian:9_. From what I talked with Faustin, the solution was to have a separate ppc64le tag on quay. Do you have any ideas on how to avoid this?

**Comment by grooverdan** [comment]
Most preferred:

Drop support for ppc64le on Debian LTS since its not supported upstream - https://wiki.debian.org/LTS. This would be consistent with our other policies about not testing/building on unsupported platforms.

**Comment by vladbogo** [comment]
From my side that is definitely an option. It seems that we already release Debian 9, so that might pose some issues. Anyway, not merging for the moment.

**Comment by fauust** [comment]
The ppc architecture is not part of the [LTS](https://wiki.debian.org/LTS) program, so I am really wondering why we support it...
Ping @cvicentiu, @dbart, @grooverdan ?

**Comment by grooverdan** [comment]
per release notes, no more debian stretch (9) - https://mariadb.com/kb/en/mariadb-10335-release-notes/

**Pull request closed without merging**